### ISSUE TITLE
Ring Keypad V2 Home Assistant

### ISSUE THREAD

**Issue by Grosscoder** [ImSorryButWho/HomeAssistantNotes#27]
How do I configure one button on the ring keypad V2 to night mode for alarmo in home assistant?

**Comment by Grosscoder** [comment]
![PXL_20231211_005016931.jpg](https://github.com/ImSorryButWho/HomeAssistantNotes/assets/91350220/b46a9842-e199-49ea-b498-2bc02654b688)

I want that one of the top buttons should be night mode.

**Comment by Idaho947** [comment]
yes, I can confirm that the night mode does not alarm like the others. However, it triggers when a sensor sees movement;
